src/core/search/search.cc

```diff
@@ -188,13 +195,18 @@ struct BasicSearch {
     if (sub_results.empty())
       return vector<DocId>{};
 
-    // Unifying from smallest to largest is more efficient.
-    // AND: the result only shrinks, so starting with the smallest is most optimal.
-    // OR: unifying smaller sets first reduces the number of element traversals on average.
+      // Unifying from smallest to largest is more efficient.
+      // AND: the result only shrinks, so starting with the smallest is most optimal.
+      // OR: unifying smaller sets first reduces the number of element traversals on average.
+
+      // Ignore Wmaybe-uninitialized because it yields maybe-uninitialized with GCC 12
```

Comment:
Indentation is off here, right? Is it a mistake?

Reply:
The empty line? I don't like having two comments on different topics merged as one.

I usually read only the first few words, if I'm not interested I skip the whole block

Reply:
No, I meant that the indentation (the number of spaces of prefix) in your proposed change is aligned with the (terminated) `if` above, no?
Before your change it matched the flow after the `if`

Reply:
Ah, true, its the auto formatter. Don't know how to fix it easily 🤷🏻‍♂️

Reply:
You could add scope brackets to the `if` maybe?
Anyway, not crucial I guess

Reply:
I removed the pragma from this function and it lined up well again